packages/airnode-validator/README.md

```diff
@@ -15,45 +15,15 @@ Validator is currently an internal dependency of Airnode and not intended to be
 You can clone and build the Airnode monorepo then run the validator as a yarn script from inside the
 `packages/airnode-validator` directory.
 
+<!-- TODO: fix commands for v2 validator cli -->
+
 ```sh
```

Comment:
Feel free to remove this section below as well. The CLI will not use specs file (the validation will be based on the zod schema).